scripts/uberenv/packages/axom/package.py

```diff
@@ -366,65 +389,83 @@ def hostconfig(self, spec, prefix):
 
         # BlueOS
         if on_blueos or on_blueos_p9:
+            familymsg = ("All of BlueOS compilers report clang "
+                         "due to nvcc, override to proper compiler family")
             if "xlf" in f_compiler:
                 cfg.write(cmake_cache_entry("CMAKE_Fortran_COMPILER_ID", "XL",
-                    "All of BlueOS compilers report clang due to nvcc, override to proper compiler family"))
+                                            familymsg))
             if "xlc" in c_compiler:
                 cfg.write(cmake_cache_entry("CMAKE_C_COMPILER_ID", "XL",
-                    "All of BlueOS compilers report clang due to nvcc, override to proper compiler family"))
+                                            familymsg))
             if "xlC" in cpp_compiler:
                 cfg.write(cmake_cache_entry("CMAKE_CXX_COMPILER_ID", "XL",
-                    "All of BlueOS compilers report clang due to nvcc, override to proper compiler family"))
+                                            familymsg))
 
             if "xlf" in f_compiler:
-                cfg.write(cmake_cache_entry("BLT_FORTRAN_FLAGS", "-WF,-C!  -qxlf2003=polymorphic",
-                    "Converts C-style comments to Fortran style in preprocessed files"))
+                description = ("Converts C-style comments to Fortran style "
+                               "in preprocessed files")
+                cfg.write(cmake_cache_entry("BLT_FORTRAN_FLAGS",
+                                            "-WF,-C!  -qxlf2003=polymorphic",
+                                            description))
                 # Grab lib directory for the current fortran compiler
-                libdir = os.path.join(os.path.dirname(os.path.dirname(f_compiler)), "lib")
+                libdir = os.path.join(os.path.dirname(
```

Comment:
Is this duplicated on purpose?

Reply:
Yes this is on purpose.  It's the equivalent to "../..".

Reply:
Thanks @white238.

Should we add a comment? Or is it obvious to everyone (except me)?

Reply:
its super duper obvious.. I will add a comment.

Reply:
hey! you almost tricked me! there is a comment!

Reply:
I meant about the double ``os.path.dirname``.

Looking at it again, it is probably sufficient as is.